src/obfl-specification.html

```diff
@@ -1574,13 +1577,34 @@ <h4 id="L1105" class="include">toc-entry</h4>
         <dt>ref-id [required]</dt>
           <dd>The id of the block this entry is connected with.</dd>
       </dl>
+    </dd>
+  <dt>Content Model</dt>
+    <dd>toc-entry-content [1 or more], toc-entry [0 or more]</dd>
+	<dd>There must be exactly one toc-entry-content element without a use-when attribute.</dd>
+</dl>
+</div>
+
+<h4 id="L1106" class="include">toc-entry-content</h4>
+
+<p>The toc-entry-content element defines the content of a toc-entry.</p>
+
+<div class="sidebar markup">
+<dl>
+  <dt>Usage</dt>
+    <dd>toc-entry</dd>
+  <dt>Attributes</dt>
+    <dd><dl>
+        <dt>use-when [optional]</dt>
+          <dd>A condition.</dd>
+		  <dd>The only currently supported condition is: $resumed.</dd>
```

Comment:
It's fine to do this in the implementation but don't put it in the specification. I would turn it around. Mention that `$resumed` is a supported variable, and explain what it is, but don't say that it is the only supported condition. For example we also discussed the possibility to say `(not $resumed)`.